modules/logging-in.adoc

```diff
@@ -0,0 +1,53 @@
+:_module-type: PROCEDURE
+
+[id='logging-in_{context}']
+= Logging in to {productname-short}
+
+[role='_abstract']
+After you install {productname-short}, log in to the {productname-short} dashboard so that you can set up your develop and deploy environment.
```

Comment:
Is 'develop and deploy' correct here, or should we have 'development and deployment'?